auth/resource.cc

```diff
@@ -11,6 +11,7 @@
 #include "auth/resource.hh"
 
 #include <algorithm>
+#include <fmt/core.h>
```

Comment:
@pwrobelse no, it's not intended. we should include it as none of the included headers in `auth/resource.hh` is supposed to provide fmt, so it's our responsibility to include by ourselves.

> Shouldn't `fmt/core.h` be included in the header file instead of the source file?

i don't think header or source file is the reason why `fmt/core.h` is included or not. IMHO what matters it's the dependency and modularity.